packages/synchro-charts/src/components/sc-widget-grid/sc-grid-tooltip.tsx

```diff
@@ -79,24 +86,30 @@ export class ScGridTooltip {
                   </div>
                 )}
 
-                {this.alarmPoint && (
-                  <div>
-                    <div class="awsui-util-label">Status:</div>
-                    <div>
-                      <strong style={{ color }}>{this.alarmPoint.y}</strong> since{' '}
-                      {new Date(this.alarmPoint.x).toLocaleString('en-US', {
-                        hour12: true,
-                        minute: 'numeric',
-                        hour: 'numeric',
-                        year: 'numeric',
-                        month: 'numeric',
-                        day: 'numeric',
-                      })}
-                      {this.breachedThreshold && this.breachedThreshold.description && (
-                        <div>({this.breachedThreshold.description})</div>
-                      )}
-                    </div>
-                  </div>
+                {label ? (
+                  <strong style={{ color }}>{this.breachedThreshold?.label?.text}</strong>
```

Comment:
will there always be a breached threshold when label is defined?

Reply:
`const label = this.breachedThreshold ? this.breachedThreshold.label : undefined;`

`label` is by the breachedThreshold defined So the threshold is always breached when label is defined.
If the label text is null, just not show the text.